httpkit/client/runtime.go

```diff
@@ -135,7 +135,14 @@ func (r *Runtime) Submit(operation *client.Operation) (interface{}, error) {
 	}
 	req.URL.Scheme = r.pickScheme(operation.Schemes)
 	req.URL.Host = r.Host
+	var reinstateSlash bool
+	if req.URL.Path[len(req.URL.Path)-1] == '/' {
```

Comment:
I think you want to check if the string is not empty before doing this. I think it panics when the url path is empty